guides/introduction.md

```diff
@@ -0,0 +1,48 @@
+# Introduction
+
+[![Build Status](https://travis-ci.org/esl/MongoosePush.svg?branch=master)](https://travis-ci.org/esl/MongoosePush) [![Coverage Status](https://coveralls.io/repos/github/esl/MongoosePush/badge.svg?branch=master)](https://coveralls.io/github/esl/MongoosePush?branch=master) [![Ebert](https://ebertapp.io/github/esl/MongoosePush.svg)](https://ebertapp.io/github/esl/MongoosePush)
+
+**MongoosePush** is a simple, **RESTful** service written in **Elixir**, providing ability to **send push
+notifications** to `FCM` (Firebase Cloud Messaging) and/or
+`APNS` (Apple Push Notification Service) via their `HTTP/2` API.
+
+# Quick start
+
+## Running from DockerHub
+
+We provide prebuilt MongoosePush images. Configuration requires either an FCM token, APNS certificates or an APNS token. Depending on your usecase, you can have some or all of them in a standalone MongoosePush instance or using a docker container.
+For the full configuration you need to set the following directory structure up:
+* priv/
+    * ssl/
+      * rest_cert.pem - The HTTP endpoint certificate
+      * rest_key.pem - private key for the HTTP endpoint certificate (has to be unencrypted)
+    * apns/
+      * prod_cert.pem - Production APNS app certificate
+      * prod_key.pem - Production APNS app certificate's private key (has to be unencrypted)
+      * dev_cert.pem - Development APNS app certificate
+      * dev_key.pem - Development APNS app certificate's private key (has to be unencrypted)
+      * token.p8 - `APNS` authentication token
+    * fcm/
+      * token.json - `FCM` service account JSON file
+
+If you want to use `APNS` token authentication you need to provide token and set `key_id` and `team_id` environmental variables. To see how to obtain token and `key_id` read:
+https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/establishing_a_token_based_connection_to_apns
+To see how to obtain `team_id` read: https://www.mobiloud.com/help/knowledge-base/ios-app-transfer/
+`FCM` JSON file can be generated by Firebase console (https://console.firebase.google.com). Go to your project -> `Project Settings` -> `Service accounts` -> `Generate new private key`
+Assuming that you have the `priv` directory with all certificates and fcm token in current directory, then you may start MongoosePush with the following command:
+
+```bash
+docker run -v `pwd`/priv:/opt/app/priv \
+  -e PUSH_HTTPS_CERTFILE="/opt/app/priv/ssl/rest_cert.pem" \
+  -e PUSH_HTTPS_KEYFILE="/opt/app/priv/ssl/rest_key.pem" \
+  -it --rm mongooseim/mongoose-push:latest
+```
+
+## Local build prerequisites
```

Comment:
And this section is also duplicated from somewhere else

Reply:
Yeah, we have to think what to put in the introduction though, as these sections make sense as they are I think